src/lib.rs

```diff
@@ -0,0 +1,956 @@
+// Copyright 2014-2015 The Servo Project Developers. See the COPYRIGHT
```

Comment:
Heh, this is an artifact of me starting to write this code in the old Servo rust-png repo.